src/viam/api/CMakeLists.txt

```diff
@@ -178,12 +181,98 @@ if (VIAMCPPSDK_USE_DYNAMIC_PROTOS)
     ${PROTO_GEN_DIR}/component/servo/v1/servo.grpc.pb.h
     ${PROTO_GEN_DIR}/component/servo/v1/servo.pb.cc
     ${PROTO_GEN_DIR}/component/servo/v1/servo.pb.h
+    ${PROTO_GEN_DIR}/google/api/annotations.grpc.pb.cc
```

Comment:
The existing inclusion of things from `google/api` appears only to have pulled in the `.pb` files and not the `grpc.pb` files. Can you see if things still work if you filter out the `grpc` ones?

Reply:
You're right: a lot of the generated protobuf files were unused and could be removed again!